### ISSUE TITLE
COPYING file

### ISSUE THREAD

**Issue by khufkens** [geco-bern/rsofun#179]
Copying copyright file to resolve issues regarding attribution and sourcing.

https://www.gnu.org/licenses/gpl-howto.html

```
Copyright 1998, 1999 Terry Jones

This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.

This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.

You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.

---

Other Statements regarding authors etc...

```

**Comment by khufkens** [comment]
Yes, will patch this. Will explain it if you are interested as well. This IP business isn't thought in "school" but part of business (and academia if you like it or not).

**Comment by khufkens** [comment]
FYI this is largely independent, so make any changes and a pull request then I'll patch this - no need to waith.

**Comment by khufkens** [comment]
Tracks here: https://github.com/geco-bern/rsofun/tree/patch-licensing

**Comment by khufkens** [comment]
Fixed per #181 - please update your fork before pushing to avoid later merge conflicts
